python/amici/ode_export.py

```diff
@@ -2372,7 +2372,8 @@ def _get_function_body(self,
                 for index, formula in enumerate(
                         self.model.eq('x0_fixedParameters')
                 ):
-                    if formula != 0 and formula != 0.0:
+                    if formula != 0 and formula != 0.0 \
```

Comment:
```suggestion
                    if any([sym in formula.free_symbols for sym in k]):
```

Reply:
alternatively the respective index set can be stored when computing `fx0_fixedParameters`